packages/schematics/angular/guard/index_spec.ts

```diff
@@ -105,4 +105,17 @@ describe('Guard Schematic', () => {
       expect(fileString).toContain(functionName);
     });
   });
+
+  it('should use CanActivate if no implements value', async () => {
+    const options = { ...defaultOptions, implements: []};
```

Comment:
`implements` - cannot be an empty array, as it's invalid. it should be `undefined`. so that the defaults are applied.